setup.py

```diff
@@ -24,15 +24,13 @@
 # along with satpy.  If not, see <http://www.gnu.org/licenses/>.
 """Setup file for satpy."""
 
-import imp
 import os.path
 import sys
 from glob import glob
+import versioneer
```

Comment:
We'll have to test that this works properly with the package when installed from PyPI. Probably does since I'm guessing you got this from the versioneer docs.

Reply:
yes, I'm doing as documented, and as dask and xarray are doing